pages/lobby/quests/[id].tsx

```
@@ -233,8 +228,38 @@ const QuestDetailsPage = () => {
           }
           break;
+        case "deploy_lock":
+        case "vendor_buy":
+        case "vendor_sell":
+        case "vendor_light_up":
+          // If inputData is an object (from specialized forms), it contains transactionHash
+          // If it's a string (from generic input), assume it's the transactionHash
+          const txHash =
+            typeof inputData === "object"
+              ? (inputData as any).transactionHash
+              : inputData;
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)